release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,56 +1,81 @@
 Bug Triage Release Team Role
 ==============================
 
-Your job is to make sure that bugs which affect the release are dealt with in a timely fashion.  Bugs can appear either as GitHub Issues or Pull Requests (PRs).  
+Primarily, your job is to make sure that bugs which affect the release are dealt with in a timely fashion.  
+Bugs can appear either as GitHub Issues or Pull Requests (PRs).  
 
-Primarily, you will be polling bugs, checking status, sending reminders and questions to contributors and SIG leads, and publishing summary reports.  The role has been interchangeably called "Bug Triage" and "Issue Triage", but the general term "issue" can be conflated with the specific GitHub usage of `is:issue` artifact type.  It is important to consider both `is:issue` and `is:pr` GitHub artifacts when triaging bugs.
+A list of common tasks is:
+- Polling for bugs (mainly issues with the label `kind/bug`)
```

Comment:
Looking for completely unlabeled issues is tricky: it can be a humongous task and I think it's a result of collective best effort and not necessarily under bug triage.

I've covered labelling / filtering in later sections (but still adding stuff, not ready for review)